### PR TITLE
Avoid flattening a 1D numpy array

### LINKED ISSUE #732: FutureWarning: Series.ravel is deprecated
When tests are run, the following warning pops up.
```
test/integration/test_model.py::test_train_with_selected_features
  /home/runner/work/aibolit/aibolit/aibolit/model/model.py:124: FutureWarning: Series.ravel is deprecated. The underlying array is already 1D, so ravel is not necessary.  Use `to_numpy()` for conversion to a numpy array instead.
    self.model.fit(X, y.ravel(), logging_level='Silent')
```
Image

### PR DESCRIPTION
**Pull request by ivanovmg**
This PR avoid flattening a 1D numpy array with `ravel` method, which emits a warning for 1D arrays.

Closes #732

### CODE REVIEW

**Comment by yegor256** [comment]
@ivanovmg cool, thanks!

**Comment by 0crat** [comment]
@ivanovmg Hey there! 👋 Thanks for your contribution! You've earned +4 points, which is our minimum reward. Here's the breakdown: +16 base points, -16 for no code review (super important!), -8 because we aim for at least 20 [hits-of-code](https://www.yegor256.com/2014/11/14/hits-of-code.html) (you had 2), and +12 to hit our minimum. Remember, more [hits-of-code](https://www.yegor256.com/2014/11/14/hits-of-code.html) and getting your code reviewed can boost your points next time! [Keep](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) the contributions coming – we're excited to see what you'll do next! 💻🚀
